packages/generate-html/test/generate-html.spec.ts

```diff
@@ -35,6 +37,18 @@ describe('GenerateHTML', function () {
       expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/EN10168/v0.2.0/template_hbs.html`, 'utf-8'),
       expectedHtmlFromMjml: '',
     },
+    {
+      type: SupportedSchemas.EN10168,
+      version: 'v0.3.0',
+      certificatePath: `${__dirname}/../../../fixtures/EN10168/v0.3.0/valid_cert.json`,
+      schemaTranslationsPath: `${__dirname}/../../../fixtures/EN10168/v0.3.0/translations.json`,
+      schemaInterface: readFileSync(`${__dirname}/../../../fixtures/EN10168/v0.3.0/certificate.ts`, 'utf-8'),
+      expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/EN10168/v0.3.0/template_hbs.html`, 'utf-8'),
+      localTemplatePath: `${__dirname}/../../../fixtures/EN10168/v0.3.0/template.hbs`,
+      localTemplatePartialPath: `${__dirname}/../../../fixtures/EN10168/v0.3.0/inspection.hbs`,
```

Comment:
`localTemplatePartialPath` should be an object instead of a string.
Also it should be usable for remote cases where partials needs to be downloaded and compiled before rendering.
Something like:

The properties should be the name of the partial as it is used in the main HBS template, in that case :
```ts
...
templatePartialPaths: { 
 // for localOnly===true we provide a file path
  inspection: `${__dirname}/../../../fixtures/EN10168/v0.3.0/inspection.hbs`,
  // for localOnly===false we  just provide the location in the server that hosts schema and static resources
  inspection: 'inspection.hbs' 
}
```